### ISSUE TITLE
Rename "beginner" labels as "introduction"

### ISSUE THREAD

**Issue by m09** [src-d/awesome-machine-learning-on-source-code#144]
As discussed in #55, we should rename beginner to introduction.

**Comment by vmarkovtsev** [comment]
So actually I think that we should revisit this, because there will be some people for sure who will say that some paper is not an introduction, they will open an issue and we will have to discuss everything again.

I would rather add a second label and see how many of each we get.

**Comment by m09** [comment]
I agree on the ambiguity of the label. Should we forgo introduction completely then? Or establish a better typology with labels such as bibliographic-review, experimential, theoretical, etc?

**Comment by bzz** [comment]
Usually, in decisions like this - to pick a better name or option what helps me is to recall what problem are we trying to solve with it.

**Comment by vmarkovtsev** [comment]
@bzz I think that the core problem was https://github.com/src-d/awesome-machine-learning-on-source-code/issues/55#issuecomment-396596568
